api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java

```diff
@@ -70,6 +70,56 @@ public void testGetOutputInfo() {
         }
     }
 
+    @Test
+    public void testSetNullInputInfo() {
+        try {
+            mSingle.setInputInfo(null);
+            fail();
+        } catch (Exception e) {
+            /* expected */
+        }
+    }
+
+    @Test
+    public void testSetInputInfo() {
+        String root = Environment.getExternalStorageDirectory().getAbsolutePath();
+        File model = new File(root + "/nnstreamer/test/add.tflite");
+
+        if (!model.exists()) {
+            fail();
+        }
+
+        SingleShot addSingle = new SingleShot(model);
```

Comment:
move this line into try-catch block to get the exception case when open model failed

Reply:
updated.